### DISCUSSION TITLE
Schema.graphql line excemption.

### DISCUSSION THREAD

**Discussion by Darbiol** [MichalLytek/type-graphql#1133] [Q&A]
Is there a way for a specific region or lines in the generated `schema.graphql` to be left untouched when generating the file?

**Comment by MichalLytek** [comment]
No, generated `schema.graphql` is 1:1 representation of your runtime schema build by TypeGraphQL.
It's only for 3rd-party tooling like GraphQL codegen and IDE extensions query autocomplete features.
